src/site/content/en/react/performance-as-a-default-with-nextjs/index.md

```diff
@@ -0,0 +1,152 @@
+
+Although Next.js provides a number of performance optimizations by default, these guides
+aim to explain them in more detail and show you how you can use them to build a fast and
+performant experience.
+
+{% Aside %}
+  This collection assumes that you have a basic knowledge of React. If not, check out 
+  [React's Getting Started page](https://reactjs.org/docs/getting-started.html).
+{% endAside %}
+
+There are many optimizations that can be added to React sites in general that would also
+work for applications built with Next.js. These will not be covered since the focus is
+on what Next.js specifically provides. To learn more about general React
+optimizations, check out our [React collection][collection].
+
+## How is Next.js different from React?
+
+React is a library that makes it easier to build user interfaces using a component-based
+approach. Although powerful, React is specifically a UI library. Many developers include
+additional tooling such as a module bundler ([webpack](https://webpack.js.org/) for example)
+and a transpiler ([Babel](https://babeljs.io/) for example) to have a complete build toolchain.
+
+In the [React collection][collection], we took the approach of using [Create React App](https://create-react-app.dev/)
+(CRA) to spin up React apps quickly. CRA takes the hassle out of setting up a React application
+by providing a complete build toolchain with a single command.
+
+Although there are a few default optimizations baked into CRA, the tool aims to provide a
+simple and straightforward setup. The choice is given to developers to decide whether
+to [eject](https://create-react-app.dev/docs/available-scripts#npm-run-eject) and
+modify the configurations themselves.
+
+Next.js, which can also be used to create a new React application, takes a different approach.
+It immediately provides a number of common optimizations that many developers would like to have
+but find difficult to set up, such as:
+
+* Server-side rendering
+* Automatic code-splitting
+* Route prefetching
+* File-system routing
+* CSS-in-JS styling (`styled-jsx`)
+
+## Setting up
+
+To create a new Next.js application, run the following command:
+
+```bash
+npx create-next-app new-app
+```
+
+{% Aside %}
+  [npx](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b)
+  is a package runner that is installed automatically with npm 5.2.0 or later. It simplifies
+  quite a few processes involved with managing packages including running CLI commands
+  (like `create-next-app`) without having to install them globally on your machine.
+{% endAside %}
+
+Then navigate to the directory and start the development server:
+
+```bash
+cd new-app
+npm run dev
+```
+
+{% Aside %}
+  You can also add Next.js to an existing React application.
+  Check out [Manual Setup](https://nextjs.org/docs#manual-setup) to learn how.
+{% endAside %}
+
+The following embed shows the directory structure of a new Next.js app. Click **Remix to
```

Comment:
If possible, I'd look into using the `Instructions` component for these steps. Glitch has a tendency to make changes to their UI and it's nice to update that component and have all dependent articles pick up the change.

Reply:
https://web.dev/handbook/web-dev-components/#instruction

Reply:
Good catch